src/main/assembly/dist/cfg/account-substitutes.csv

```diff
@@ -0,0 +1,2 @@
+removed-account, chosen-account
```

Comment:
it would be nice if the absence of this file is not breaking

Reply:
Done:
https://github.com/DANS-KNAW/easy-convert-bag-to-deposit/blob/2cc92759b0cd276c8bb96a14680957eecf1bcf54/src/main/scala/nl.knaw.dans.easy.bag2deposit/UserTransformer.scala#L29-L30